schema/data/dev/application-2018.sql

```diff
@@ -1,12 +1,7 @@
 begin;
 
--- Set the current reporting year to 2018
-create or replace function ggircs_portal.current_timestamp() returns timestamptz as
-$$
-  select application_open_time
-  from ggircs_portal.reporting_year
-  where reporting_year = 2018
-$$ language sql;
+  -- Set the current reporting year to 2018, application open
+  select mocks.set_mocked_time_in_transaction('2019-04-01 14:49:54.191757-07'::timestamptz);
```

Comment:
any reason why the time is hardcoded here instead of 
```suggestion
  select mocks.set_mocked_time_in_transaction(
    select application_open_time
      from ggircs_portal.reporting_year
      where reporting_year = 2018
  );
```

Reply:
Good point, I pasted this similarly to our db unit tests